r/configure

```diff
@@ -32,13 +32,26 @@ PKG_RPM_NAME="arrow"
 PKG_CSW_NAME="arrow"
 PKG_BREW_NAME="apache-arrow"
 PKG_TEST_HEADER="<arrow/api.h>"
-PKG_LIBS="-larrow -lparquet"
+PKG_LIBS=""
 
 # Use pkg-config if available
 pkg-config --version >/dev/null 2>&1
 if [ $? -eq 0 ]; then
   PKGCONFIG_CFLAGS=`pkg-config --cflags --silence-errors ${PKG_CONFIG_NAME}`
   PKGCONFIG_LIBS=`pkg-config --libs ${PKG_CONFIG_NAME}`
```

Comment:
Needless codes are remained.
I've created a pull request: #3951

Reply:
It's not needless code, it's a default on systems where `pkg-config` is unavailable. Also this PR has broken the build on MacOS. Did anyone actually test this?

Reply:
Could you open a new JIRA issue for this?